src/classes/CW_AdminController.cls

```diff
@@ -0,0 +1,168 @@
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 17/04/2014 Chris Yorkston Initial version
+* 0.2 14/05/2014 AleX Panka     Adding Uniqa groups
+*
+*
+*/
+
+
+	private List<CronTrigger> allCronJobs;
+
+	public Id batchId {get; set;}
+	public Boolean hasCronJob {
+		get {
+			return getCronJobs().Size()>0;
+		}
+		set;
+	}
+
+	public List<CronTrigger> getCronJobs(){
+		List<CronTrigger> resLst= new List<CronTrigger>();
+		for(CronTrigger rec: allCronJobs){
+			if(rec.CronJobDetail.Name.StartsWith(ConstantUtil.BATCH_APEX_JOB_NAME_UPDATE_AUDIENCE_GROUP_MEMBER))
+				resLst.Add(rec);
+		}
+		return resLst;
+	}
+
+
+//Uniqa gruop Members params
+	public String CntrGrpSuffix{get;set;}
+	public String TrgtGrpSuffix{get;set;}
+	public String batchIdUniqa{get;set;}
+	public Boolean getHasUniqaCronJob() {
+		return getUniqacronJobs().Size()>0;
+	}
+	public List<CronTrigger> getUniqacronJobs(){
+		List<CronTrigger> resLst= new List<CronTrigger>();
+		for(CronTrigger rec: allCronJobs){
+			if(rec.CronJobDetail.Name.StartsWith(ConstantUtil.BATCH_APEX_JOB_NAME_UPDATE_AUDIENCE_GROUP_RELATION))
+				resLst.Add(rec);
+		}
+		return resLst;
+	}
+
+   public Date dateCustomSetting {get;set;}
+
+	public CW_AdminController() {
+	    ReloadCronJobs();
+	    dateCustomSetting = CustomSettingUtil.getRunBatchDate();
+        CntrGrpSuffix=CW_CampaignListSettingHelper.TextValue(ConstantUtil.AUDIENCE_CONTROL_GROUP_SUFFIX_NAME,ConstantUtil.AUDIENCE_CONTROL_GROUP_SUFFIX_VALUE);
+        TrgtGrpSuffix=CW_CampaignListSettingHelper.TextValue(ConstantUtil.AUDIENCE_TARGET_GROUP_SUFFIX_NAME , ConstantUtil.AUDIENCE_TARGET_GROUP_SUFFIX_VALUE);
+
+	}
+
+	public Pagereference updateAudienceGroupMembers() {
+
+		Id batchInstanceId = Database.executeBatch(new CW_BatchUpdateAudienceGroupMember(), 2000);
+		batchId = batchInstanceId;
+
+		return null;
+	}
+
+	public Pagereference addScheduledJobToUpdateAudienceGroupMembers() {
+
+		// Instantiate the batch class
+		CW_BatchUpdateAudienceGroupMember myBatch = new CW_BatchUpdateAudienceGroupMember();
+
+		// Instantiate the scheduler
+		BatchScheduler scheduler = new BatchScheduler();
+
+		// Assign the batch class to the variable within the scheduler
+		scheduler.batchClass = myBatch;
+
+		// Get Schedule string - run every day at 1am (01:00)
+		String sch = ConstantUtil.BATCH_APEX_SCHEDULE_UPDATE_AUDIENCE_GROUP_MEMBER;
+
+ 		// schedule batch apex job
+		System.schedule(ConstantUtil.BATCH_APEX_JOB_NAME_UPDATE_AUDIENCE_GROUP_MEMBER+' - Everyday at 1pm', sch, scheduler);
+
+		// query cron jobs
+		ReloadCronJobs();
+		return null;
+	}
+
+	public Pagereference removeScheduledJobToUpdateAudienceGroupMembers() {
+
+		// delete scheduled job
+		for (CronTrigger jobs :getCronJobs()) {
+			System.abortJob(jobs.Id);
+		}
+
+		// query cron jobs
+		ReloadCronJobs();
+		return null;
+	}
+
+	public void  ReloadCronJobs() {
```

Comment:
Methods should use the `lowerCamelCase` naming convention. I suggest rename accordingly.